README.md

```diff
@@ -26,14 +26,20 @@ Have a clean repository that meets quality coding standards (file structure, nam
 Have a quality README (with unique name, description, technologies used, screenshot, and link to deployed application).
 
 ## Web Application Preview
-![Main screen](<Screenshot 2023-10-08 231308.png>)
-![Random movie chosen](<Screenshot 2023-10-08 231622.png>)
-![Youtube trailer](<Screenshot 2023-10-08 231637.png>)
+![Main screen](<webpage-preview-1>)
```

Comment:
@itsbbea I think you need to add the path for these webpage-preview links so something like
assets/images/webpage-preview-1 for the images to appear

Currently looks like this:
![Screenshot 2023-10-09 at 5 45 49 pm](https://github.com/ajayshans/moviequest/assets/69303149/c4e5ae84-bfa7-4a09-ae2a-7b4825480aef)